<?xml version="1.0"?>
<launch>
  <arg name="model_path"   default="$(find ffld_nx)/data/models/towne_redchair.txt"/>
  
  <node name="object_detector" pkg="ffld_ros" type="detect_object_server"
        output="screen">
    <param name="model_name" value="$(arg model_path)"/> 
    <param name="overlap" value="0.5" />    <!-- Minimum overlap in non maxima suppression (default 0.5)  -->
    <param name="threshold" value="-0.5" />  <!-- Minimum detection threshold (default -1) -->
  </node>
</launch>

